init/updateboot.pl

```diff
@@ -20,7 +20,14 @@
 			unlink("$p/$product.service");
 			unlink("$p/$product");
 			}
-		copy_source_dest("../webmin-systemd", "$systemd_root/$product.service");
+		my $temp = &transname();
+		&copy_source_dest("../webmin-systemd", "$temp");
```

Comment:
Use an absolute path rather than ..

Reply:
I still don't see why an absolute path like `$root_directory/webmin-systemd` can't be used here?